### PR TITLE
feat(pagination): add horizontal alignment config option

### PR DESCRIPTION
**Pull request by tmeneau**
This is similar to pull request #1210 -- except applied to the pagination component -- so I used the same configuration name (`justify`) and configurable values for consistency: 

* 'start' (default)
* 'center'
* 'end'

### CODE REVIEW

File: src/pagination/pagination-config.ts [hunk 1]
```diff
@@ -15,4 +15,5 @@ export class NgbPaginationConfig {
   pageSize = 10;
   rotate = false;
   size: 'sm' | 'lg';
+  justify: 'start' | 'center' | 'end' = 'start';
```
Comment (maxokorokov) [inline src/pagination/pagination-config.ts]: Please order alphabetically

File: src/pagination/pagination.ts [hunk 2]
```diff
@@ -109,6 +109,11 @@ export class NgbPagination implements OnChanges {
    */
   @Input() size: 'sm' | 'lg';
 
+  /**
+   * Horizontal positioning with Flex justification: start, center, or end
+   */
+  @Input() justify: 'start' | 'center' | 'end';
```
Comment (maxokorokov) [inline src/pagination/pagination.ts]: Please order alphabetically

**Comment by tmeneau** [comment]
Thanks for the review! I went ahead and corrected the ordering in the:

* component members
* configuration members
* spec references and test classes

**Comment by pkozlowski-opensource** [comment]
@tmeneau do you have a real-life use-case where just applying Bootstrap utility classes isn't enough? Asking since I can already position pagination control without any changes to this library: http://plnkr.co/edit/vIzZoSE6auPfrTBm4rxG?p=preview

IMO pagination use-case is different as compared to the tabset one (in a tabset we want to justify tabs to which we don't have separate access but in pagination we can take the entire control and move it around...).

**Comment by tmeneau** [comment]
@pkozlowski-opensource Well, I'm a moron; thanks for taking the time to put together the plunker, I'd somehow missed that the pagination container isn't a flexbox container by default. You're completely right, I can't imagine any use case not already covered as far as pagination alignment is concerned.

Sorry about that! I'll close this pull request.

**Pull request closed without merging**